src/components/Datepicker/Datepicker.tsx

```diff
@@ -325,4 +355,6 @@ export const Datepicker: FC<DatepickerProps> = ({
   );
 };
 
+export const Datepicker = forwardRef(DatepickerRender);
```

Comment:
I'm using the forwrdRef call separatelly, as having it together with the DatepickerRender adds unnecessary indentation.

Reply:
agreed